frida_mode/src/instrument.c

```diff
@@ -0,0 +1,148 @@
+#include "frida-gum.h"
+#include "config.h"
+#include "prefetch.h"
+
+extern uint8_t *        __afl_area_ptr;
+static __thread guint64 previous_pc;
+
+GumAddress current_log_impl;
+uint64_t   afl_prev_loc;
+
+#if GUM_NATIVE_CPU == GUM_CPU_AMD64 && defined(WITH_INLINE_INSTR)
```

Comment:
We can copy&paste aarch64 support from libafl Frida here

Reply:
If @s1341 agree

Reply:
Absolutely. There's a good few more features to add yet too. Deferred fork mode and persistent mode. I just wanted to get something in as a starting point for discussion and to check everyone was happy before I wrote a whole load of code.

I'm happy to add it to the PR though if you'd like?

Reply:
Of course I agree! Note that my code is not yet optimized...